--- makefile	2023-05-18 01:29:56.788823349 +0000
+++ makefile.patched	2023-05-18 01:32:05.958683068 +0000
@@ -73,7 +73,7 @@
 	@echo "installing ly"
 	@install -dZ ${DESTDIR}/etc/ly
 	@install -DZ $(BIND)/$(NAME) -t ${DESTDIR}/usr/bin
-	@install -DZ $(RESD)/config.ini -t ${DESTDIR}/etc/ly
+	@install -DZ $(RESD)/config.ini -m 644 -t ${DESTDIR}/etc/ly
 	@install -DZ $(RESD)/xsetup.sh -t $(DATADIR)
 	@install -DZ $(RESD)/wsetup.sh -t $(DATADIR)
 	@install -dZ $(DATADIR)/lang
@@ -87,7 +87,7 @@
 	@install -DZ $(RESD)/xsetup.sh -t $(DATADIR)
 	@install -DZ $(RESD)/wsetup.sh -t $(DATADIR)
 	@install -dZ $(DATADIR)/lang
-	@install -DZ $(RESD)/lang/* -t $(DATADIR)/lang
+	@install -DZ $(RESD)/lang/* -m 644 -t $(DATADIR)/lang
 	@install -DZ $(RESD)/pam.d/ly -m 644 -t ${DESTDIR}/etc/pam.d

 installsystemd:
